app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,21 @@
+class LikesController < ApplicationController
+  def new
+    @like = Like.new
+    @user = current_user
+  end
+
+  def create
+    @post = Post.find(params[:post_id] || params[:id])
+    @like = Like.new(post_id: @post.id, author_id: current_user.id)
+    @like.author_id = current_user.id
+
```

Comment:
Optional.
While creating a new like, you are passing the post id and author id as params, so it is unnecessary to explicitly set the author id to current_user.id again.

Reply:
Thanks for mentioning